actors/builtin/power/power_test.go

```diff
@@ -1323,6 +1362,38 @@ func (h *spActorHarness) expectTotalPowerEager(rt *mock.Runtime, expectedRaw, ex
 	assert.Equal(h.t, expectedQA, qualityAdjPower)
 }
 
+/// at the start of every onEpochTickEnd, cron should get these values- they're necessary for ConfirmSectorProofsValid but don't change, so they only need to be looked up once.
+/// should expect these two sends in the tests and mock up the values as needed...
+func expectQueryNetworkInfo(rt *mock.Runtime, h *spActorHarness) {
+	st := getState(rt)
+
+	currentPower := power.CurrentTotalPowerReturn{
+		QualityAdjPowerSmoothed: st.ThisEpochQAPowerSmoothed,
+	}
+
+	currentReward := reward.ThisEpochRewardReturn{
+		ThisEpochBaselinePower:  h.thisEpochBaselinePower,
+		ThisEpochRewardSmoothed: h.thisEpochRewardSmoothed,
+	}
+	rt.ExpectSend(
+		builtin.RewardActorAddr,
+		builtin.MethodsReward.ThisEpochReward,
+		nil,
+		big.Zero(),
+		&currentReward,
+		exitcode.Ok,
+	)
+
+	rt.ExpectSend(
```

Comment:
this will go away when removing power actor query

Reply:
this is all done in the next PR fyi so im just gonna leave it for now